backend/document_store.py

```diff
@@ -0,0 +1,29 @@
+# fertiscan/io/document_storage.py
+
+import os
+
+class DocumentStore:
+    def __init__(self):
+        self.document = []
+
+    def add_page(self, page):
+        if isinstance(page, str):
+            if not os.path.exists(page):
+                raise FileNotFoundError(f"The file {page} does not exist.")
+            file = open(page)
+            image = file.read()
+            file.close()
```

Comment:
Try/Except would be more suited in this case.
> You should do what you expect to work. If it doesn’t work and an exception happens, then just catch the exception and handle it appropriately.
[Source](https://realpython.com/python-lbyl-vs-eafp/#:~:text=The%20%E2%80%9CEasier%20to%20Ask%20Forgiveness%20Than%20Permission%E2%80%9D%20(EAFP)%20Style,-Grace%20Murray%20Hopper&text=It%20suggests%20that%20right%20away,exception%20and%20handle%20it%20appropriately.&text=In%20Python%2C%20the%20EAFP%20coding%20style%20is%20pretty%20popular%20and%20common.)